### ISSUE TITLE
Unable to sort by IV or anything other than the default value on OS X 10.11

### ISSUE THREAD

**Issue by adisor19** [vinnymac/PokeNurse#190]
Steps : 

1) Sort by number of pokemons. (works)
2) Expand a species.(works)
3)Try and sort by IV or anything other than the default value and it fails on Mac OS 10.11 but works fine on macOS 10.12.

**Comment by YesThatAllen** [comment]
Until there's a chance for an update, sorting inside a species works so long as you sort the master list by `sprite` (I use the search box at the top to limit what's shown, making it easier to see the pokemon I want to evaluate)

(see also https://github.com/vinnymac/PokeNurse/issues/101#issuecomment-280889137)

**Comment by vinnymac** [comment]
I can go through the diff and see if I can find what caused this bug. Also you can't sort by sprite, but you can sort by Pokedex #, I assume this is what you are referring to.

It is weird that it would work on one OS but not the other, that is extremely odd to me. Perhaps it is just that it works under certain circumstances and not others.

**Comment by YesThatAllen** [comment]
I see.. the indicator looks like it's on "Sprite" to me.

![screenshot_4196](https://cloud.githubusercontent.com/assets/2879972/23370801/7d792406-fcdb-11e6-93c1-b104d731f8ad.png)

As it is, I am unable to sort within a species unless the master list is sorted by pokedex number.

That's true for me on my macOS 10.12 and one running 10.10 (I don't have a 10.11 one to test on at the moment)

**Comment by vinnymac** [comment]
I am not sure what change broke this, but I can now tell that it is caused by sorting the monsters.species array. We are likely using that array to keep sort order for all the species, which we are using elsewhere to access for species data. Instead of being able to quickly find a species information we would need to iterate over every single pokemon which is much slower :/ Separating sort order + the data would be nice here.

So basically by sorting the table the wrong pokemon are placed in a position, for example if I sort by name instead of Zubat being in index 40, Kartana is. So the sorting still occurs, but on the completely incorrect 'mon.

**Comment by YesThatAllen** [comment]
Confirmed working in develop. Awesome!
